alca-app/src/theme/MUITheme.js

```diff
@@ -53,7 +55,13 @@ export const theme = createTheme({
         MuiCssBaseline: {
             styleOverrides: `
                 body {
-                    background: radial-gradient(81.56% 81.56% at 50% 95.8%, #001740 0%, ${CustomPaletteColors.dark.main} 100%);
+                    background: url(
```

Comment:
Remember this file might be coming from an external source later. There are two approaches, giving a style to the body outside the theme or (later) overriding the theme object with customizations from the calling repo. I don't have a reason to choose any of those, both can fit. But just bear that in mind.

Reply:
This is a good point, and we should decide how we want to handle overrides, I imagine we can just import the theme object somewhere that we standardize in each app for any potential local customization we need to make to that object?

Open to other thoughts on it as well.

Reply:
For body customization I think the best way would be overriding the mui-provider theme locally unless we want to use the alca-app/src/style/index.css for that.